crates/sui-core/src/consensus_adapter.rs

```diff
@@ -317,10 +323,14 @@ impl ConsensusAdapter {
     /// when system operates normally.
     ///
     /// The function returns the position of this authority when it is their turn to submit the transaction to consensus.
-    fn submission_position(&self, committee: &Committee, tx_digest: &TransactionDigest) -> usize {
+    fn submission_position(
```

Comment:
Could you please add a test for this method? I believe we should have some coverage as those are quite core components that we need to ensure won't accidentally break from future refactoring